uiowa/uiowa
# Neuroscience article migration

Resolves #4260 

# How to test

* Pull a current local copy of the neuroscience.grad.uiowa.edu database.
* `blt ds --site neurosicence.grad.uiowa.edu`
* `drush @gradneuroscience.local en gradneuroscience_migrate`
* `drush @gradneuroscience.local uli admin/config/sitenow/sitenow-migrate`
* Enter details for migration using DevDesktop settings indicated in help text. Database name is 'neuroscience'.
* `drush @gradneuroscience.local ms` to see migration information
* `drush @gradneuroscience.local mim --all` to run all migrations
* Vist https://gradneuroscience.local.drupal.uiowa.edu/admin/content and filter to articles to spot check results.
